package.json

```diff
@@ -1,45 +1,45 @@
 {
-  "name": "fb_editor",
-  "private": true,
-  "engines": {
-    "node": "^16.20.1"
-  },
-  "scripts": {
-    "jstest": "mocha --recursive"
-  },
-  "dependencies": {
-    "@hotwired/stimulus": "^3.2.2",
-    "@hotwired/turbo-rails": "^7.3.0",
-    "@rails/actioncable": "^7.0.4",
-    "@rails/activestorage": "^7.0.4",
-    "@rails/ujs": "^7.0.4",
-    "@rails/webpacker": "^5.4.4",
-    "@sentry/browser": "^7.28.1",
-    "@ungap/custom-elements": "^1.3.0",
-    "@x-govuk/marked-govspeak": "^0.2.0",
-    "accessible-autocomplete": "^2.0.4",
-    "chai-dom": "^1.11.0",
-    "dompurify": "^3.0.3",
-    "govuk-frontend": "4.4.1",
-    "govuk-markdown": "^0.3.0",
-    "jquery": "^3.6.4",
-    "jquery-ui-dist": "^1.13.2",
-    "marked": "4.3.0",
-    "resolve-url-loader": "^5.0.0",
-    "sanitize-html": "^2.8.1",
-    "showdown": "^2.1.0",
-    "sinon": "^15.0.3",
-    "tabbable": "^6.0.1"
-  },
-  "version": "0.1.0",
-  "devDependencies": {
-    "@sentry/webpack-plugin": "^1.20.0",
-    "chai": "^4.3.7",
-    "form-data": "^4.0.0",
-    "jquery-ui": "^1.13.2",
-    "jsdom": "^20.0.3",
-    "jsdom-global": "^3.0.2",
-    "mocha": "^10.2.0",
-    "sinon-chai": "^3.7.0"
-  }
+    "name": "fb_editor",
```

Comment:
Unintentional indentation maybe? Can't really see if something else has changed!

Reply:
This is probably becasue I made a manual edit and my editro has reformatted on save (to use 4 spaces apparently, for some reason)